crates/solver/src/main.rs

```diff
@@ -280,6 +280,19 @@ struct Arguments {
     /// but at the same time we don't restrict solutions sizes too much
     #[clap(long, env, default_value = "15000000")]
     simulation_gas_limit: u128,
+
+    /// In order to protect against malicious solvers, the driver will check that settlements prices do not
+    /// exceed a max price deviation compared to the external prices of the driver, if this optional value is set.
+    /// The max deviation value should be provided as a percentage value. E.g. for a max price deviation
+    /// of 3%, one should set it to 3u64
```

Comment:
Not sure if that is fine grained enough.
Alternatively one could try to use `f64` or if you want to keep the properties of integers make it so that `1u64` is equal to `0.01%` or something like that.